packages/common/react-components/src/components/Table/Table.tsx

```diff
@@ -74,11 +74,14 @@ export const Table = <T extends {}>({ columns, data = [], slots = {}, onSelect,
         <div className='thead sticky top-0'>
           {headerGroups.map((headerGroup) => (
             // eslint-disable-next-line react/jsx-key
-            <div {...headerGroup.getHeaderGroupProps()} className='tr h-[2.5rem] border-b border-zinc-300'>
+            <div {...headerGroup.getHeaderGroupProps()} className='tr h-[2rem] border-b border-zinc-300'>
```

Comment:
2rem is 32px, which is one of the density sizes… should this use one of those or remain arbitrary?

`import { fineBlockSize } from '../../styles/density'`